locations/mlk-nk.yml

```diff
@@ -53,19 +53,15 @@ networks:
     name: mesh_nno_5
     prefix: 10.31.69.33/32
     ipv6_subprefix: -11
-    mesh_metric: 1024
     mesh_metric_lqm: ['default 0.6']
-    ptp: true
 
   # Nanostation M5 - Airos 6 - Orientation Sonnenallee
   - vid: 12
     role: mesh
     name: mesh_so_5
     prefix: 10.31.69.34/32
     ipv6_subprefix: -12
-    mesh_metric: 1024
     mesh_metric_lqm: ['default 0.7']
-    ptp: true
```

Comment:
see above

Reply:
The device is configured in AP mode therefore it can be ptmp.